clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fdg_pet.py

```diff
@@ -2,9 +2,72 @@
 
 """Module for converting FDG PET of ADNI."""
 
+PREPROCESSING_STEP_DICT = {
+    0: ["ADNI Brain PET: Raw FDG"],
+    1: ["Co-registered Dynamic"],
+    2: ["Co-registered, Averaged"],
+    3: ["Coreg, Avg, Standardized Image and Voxel Size"],
+    4: ["Coreg, Avg, Std Img and Vox Siz, Uniform Resolution"],
+    5: ["Coreg, Avg, Std Img and Vox Siz, Uniform 6mm Res"],
+}
+
 
 def convert_adni_fdg_pet(
     source_dir, csv_dir, dest_dir, conversion_dir, subjs_list=None, mod_to_update=False
+):
+    """Convert "Co-registered, Averaged" FDG PET images of ADNI into BIDS format.
+
+    Args:
+        source_dir: path to the ADNI directory
+        csv_dir: path to the clinical data directory
+        dest_dir: path to the destination BIDS directory
+        conversion_dir: path to the TSV files including the paths to original images
+        subjs_list: subjects list
+        mod_to_update: If True, pre-existing images in the BIDS directory will be erased and extracted again.
+    """
+    return convert_adni_fdg_pet_generic(
+        source_dir,
+        csv_dir,
+        dest_dir,
+        conversion_dir,
+        preprocessing_step=2,
+        subjs_list=subjs_list,
+        mod_to_update=mod_to_update,
+    )
+
+
+def convert_adni_fdg_pet_uniform(
```

Comment:
Since both these functions are partial application of the generic one with `preprocessing_step` set to a certain value, then better use [functools.partial](https://docs.python.org/3/library/functools.html#functools.partial) and write the docs and implementation once.